docs/build-with-0g/compute-network/sdk.md

```diff
@@ -0,0 +1,253 @@
+---
+id: sdk
+title: SDK
+sidebar_position: 3
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+You can integrate the services provided by our provider into their own products using our SDK.
```

Comment:
Developers can integrate AI inference services offered by providers on the 0G Compute Network into their own applications using the 0G SDK. To begin, only inference services are available, which can be used for tasks like running Large Language Models (LLMs) inferences and more. Storage and other features will be added in the future.

The SDK ensures a straightforward process to connect with and utilize these services efficiently.